packtools/sps/validation/journal_meta.py

```diff
@@ -69,65 +69,79 @@ def validate_issn(self, issns_dict):
-                'message': 'Got <issn pub-type="{}">{}</issn> expected <issn pub-type="{}">{}</issn>'.format(
-                    tp, issn_obtained, tp, issn_expected
-                ),
-                'advice': None if is_valid else 'Provide an ISSN value as expected: <issn pub-type="{}">{}</issn>'.format(
-                    tp, issn_expected),
-            }
+            yield format_response(
+                title='Journal ISSN',
+                parent='article',
+                parent_id=None,
+                parent_article_type=self.xmltree.get("article-type"),
+                parent_lang=self.xmltree.get("{http://www.w3.org/XML/1998/namespace}lang"),
+                item='issn',
+                sub_item='pub-type',
+                validation_type='value',
+                is_valid=is_valid,
+                expected='<issn pub-type="{}">{}</issn>'.format(tp, issn_expected),
+                obtained='<issn pub-type="{}">{}</issn>'.format(tp, issn_obtained),
+                advice='Mark ISSN value with <journal-meta><issn pub-type="{}">{}</issn>'.format(tp, issn_expected),
+                data=self.journal_issns.data,
+                error_level=error_level,
+            )
 
 
 class AcronymValidation:
     def __init__(self, xmltree):
         self.xmltree = xmltree
         self.journal_acronym = Acronym(xmltree)
 
-    def acronym_validation(self, expected_value):
+    def acronym_validation(self, expected_value, error_level="CRITICAL"):
         if not expected_value:
             raise ValidationJournalMetaException('Function requires a value to acronym')
         is_valid = self.journal_acronym.text == expected_value
-        return [
-            {
-                'title': 'Journal acronym element validation',
-                'xpath': './/journal-meta//journal-id[@journal-id-type="publisher-id"]',
-                'validation_type': 'value',
-                'response': 'OK' if is_valid else 'ERROR',
-                'expected_value': expected_value,
-                'got_value': self.journal_acronym.text,
-                'message': 'Got {} expected {}'.format(self.journal_acronym.text, expected_value),
-                'advice': None if is_valid else 'Provide an acronym value as expected: {}'.format(expected_value)
-            }
-        ]
+        yield format_response(
+            title='Journal acronym',
+            parent='article',
+            parent_id=None,
+            parent_article_type=self.xmltree.get("article-type"),
+            parent_lang=self.xmltree.get("{http://www.w3.org/XML/1998/namespace}lang"),
+            item='journal-id',
+            sub_item='@journal-id-type="publisher-id"',
+            validation_type='value',
+            is_valid=is_valid,
+            expected=expected_value,
+            obtained=self.journal_acronym.text,
+            advice='Mark journal acronym value with <journal-meta><journal-id journal-id-type="publisher-id">{}</journal-id>'.format(expected_value),
```

Comment:
@Rossi-Luciano 
`Mark journal acronym value with <journal-meta><journal-id journal-id-type="publisher-id">{}</journal-id>`
troque por
`Mark journal acronym value with <journal-id journal-id-type="publisher-id">{}</journal-id> in <journal-meta>`

Reply:
OK, @robertatakenaka